deploy/olm-catalog/manifests/assisted-service-operator.clusterserviceversion.yaml

```diff
@@ -400,7 +422,7 @@ spec:
                 - name: INSTALLER_IMAGE
                   value: quay.io/ocpmetal/assisted-installer:latest
                 - name: OPENSHIFT_VERSIONS
```

Comment:
Should this be removed?

Reply:
I remember @mhrivnak having the opinion that we shouldn't require the user to specify `OSImages` on the AgentServiceConfig.

My personal opinion is that, given it is included in the `alm-examples` such that when a user goes to Create an AgentServiceConfig it is populated with the appropriate defaults, we could remove the `OPENSHIFT_VERSIONS` from the clusterserviceversion and require it on the AgentServiceConfig.

If we remove this, then the `OSImages` must become a required field.

Reply:
Ah okay, I forgot that this would define the default. Yeah, this is fine :+1: